CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # TimescaleDB Changelog
 
+## 0.3.0 (2017-07-31)
+
+**High-level changes**
+* "Upserts" are now supported via normal `ON CONFLICT DO UPATE` syntax
```

Comment:
UPATE -> UPDATE. Maybe add note about caveat that we do not yet support ON UPDATE ON CONSTRAINT?